.node-version

```diff
@@ -0,0 +1 @@
+12.13.0
```

Comment:
@MikeRogers0 I just added the version I had installed, let me know what version you use and I'll update it to be that. Just wanted something in here :)